src/components/BottomSheet/BottomSheetLabel.js

```diff
@@ -33,9 +33,34 @@ const BottomSheetLabel = ({ variant, children, ...restProps }) => {
           color={colors.positive.base}
         />
       );
+    case "accent":
```

Comment:
Maybe that switch-case statement could be replaced with simple if -> return; return;
All StatusBadges just have different color, so it could be implemented smth like that:
```
const colorsMap = {
  alert: colors.alert.base,
  ...
  neutral: colors.onyx.light
};

if (colorsMap[variant]) {
  return (
    <StatusBadge
      {...restProps}
      label={children}
      color={colorsMap[variant]}
    />
  )
}

return (
  <LabelText {...restProps} primary>
    {" "}
    {children}
  </LabelText>
);
```